lib/include.js

```diff
@@ -175,6 +175,83 @@ Inclusion.include = function (objects, include, options, cb) {
     cb && cb(err, objects);
   });
 
+  // Find the limit of items for `inq`
+  var inqLimit = 256;
+  if(self.dataSource && self.dataSource.settings &&
+    self.dataSource.settings.inqLimit) {
+    inqLimit = self.dataSource.settings.inqLimit;
+  }
+  /**
```

Comment:
Nitpick: please add an empty line as a visual delimiter between the code block and the unrelated jsdoc comment.